dist/src/main/java/io/camunda/operate/IndexController.java

```diff
@@ -0,0 +1,30 @@
+package io.camunda.operate;
+
+import jakarta.servlet.ServletContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class IndexController {
+
+  @Autowired private ServletContext context;
+
+  @GetMapping("/index.html")
+  public String index() {
+    return "redirect:/operate";
```

Comment:
we should check if Tasklist standalone mode is used, in such case we need to redirect to /tasklist